components/brave_rewards/browser/rewards_service_observer.h

```diff
@@ -6,7 +6,9 @@
 #define BRAVE_BROWSER_PAYMENTS_PAYMENTS_SERVICE_OBSERVER_H_
 
 #include "brave/components/brave_rewards/browser/wallet_properties.h"
```

Comment:
origin/master also includes `#include "base/observer_list_types.h"` for base::CheckedObserver.

Reply:
Fixed! (it still compiled and worked without that?)